packages/radio-button/radio-button.scss

```diff
@@ -127,9 +136,8 @@ $radio-button-disabled-color--inverted: $gra-60;
         margin-right: $component-spacing--small;
 
         border-radius: 50%;
-        box-shadow: inset 0 0 0 rem(1px) currentColor, 0 0 0 0 transparent;
-        background-color: $radio-button-background-color;
-        background-color: var(--radio-button-background-color);
+        box-shadow: inset 0 0 0 rem(1px) $stein, 0 0 0 0 transparent;
+        box-shadow: inset 0 0 0 rem(1px) var(--radio-button-default-color), 0 0 0 0 transparent;
```

Comment:
Samme her som for checkbox, blir ikke rød farge ved feil

Reply:
Takker!